h2o-algos/src/main/java/water/rapids/RapidsInit.java

```diff
@@ -0,0 +1,11 @@
+package water.rapids;
+
+import water.rapids.prims.word2vec.AstWord2VecToFrame;
+
+public class RapidsInit {
+
+  public static void registerAlgoRapids() {
+    Env.init(new AstWord2VecToFrame());
```

Comment:
yes, agree - we need to connect registration of rapids operations with registration of the algo.